.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,13 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\client",
```

Comment:
Is it normal for these paths to take two backslashes "\\"?

Reply:
yes, that is part of the syntax